packages/mongodb-rag-core/src/VerifiedAnswer.ts

```diff
@@ -0,0 +1,26 @@
+import { z } from "zod";
+import { Reference } from "./services/conversations";
+import { VectorStore } from "./VectorStore";
+
+export const Question = z.object({
```

Comment:
nit on naming: for the zod types, we've been using the convention of naming them `ThingSchema`. Following that, this should be `QuestionSchema`

same for `VerifiedAnswer`, should be `VerifiedAnswerSchema`.

open to changing that convention, but just wanted to call out

Reply:
That ain't right. We should match the export value and type name, then it becomes one import and can be used either way. Really smooth! I did use Schema in one place where the type actually was a Schema within a file... otherwise, I see some usage but more instances without -Schema.